src/components/HeroHome/index.js

```diff
@@ -1,22 +1,27 @@
 import React from 'react';
 
-const HeroHome = ({ imageUrl, imageTitle, preheader }) => (
-  <div
-    className="coa-HeroHome"
-    style={{
-      backgroundImage: `
-        linear-gradient(rgba(36, 11, 51, .3), rgba(36, 11, 51, .3)),
-        url(${imageUrl})
-      `,
-      backgroundSize: 'cover',
-      backgroundPosition: '50%',
-    }}
-    role="img"
-    aria-label={imageTitle}
-  >
-    <div className="container-fluid wrapper">
-      <span className="coa-HeroHome-preheader">{preheader}</span>
-      <h2 className="coa-HeroHome-header">Austin, TX</h2>
+const HeroHome = ({ imageFilename, imageTitle, preheader }) => (
+  <div className="coa-HeroHome-container">
+    <img srcset={`${imageFilename}.width-640.jpg 640w,
```

Comment:
Seems like this srcset code will repeat pretty frequently. What do you think about turning this into an Img React Component that consumes a filename/filepath to generate this code.